libcaf_core/src/logger.cpp

```diff
@@ -55,12 +54,35 @@ constexpr const char* log_level_name[] = {
   "TRACE"
 };
 
+#ifndef CAF_MSVC
```

Comment:
This code is copy-pasted from caf_test. Please wrap this in a header (`libcaf_core/caf/color.hpp`) and remove the dups. Also, I think we should simply set all colors to `""` on MSVC and get rid of all other `#ifdef CAF_MSVC`. Paying two `<< ""` on Windows isn't a big deal and is easily avoided by setting the terminal to `UNCOLORED`.